### PR TITLE
RELATED: RAIL-3471 plugin bundle size debugging

### PR DESCRIPTION
**Pull request by no23reason**
Adds support for plugin bundle size debugging.
As part of the investigation/debugging process I upgraded webpack and to do it I had to bump both rush and pnpm (there were installation errors otherwise).
Also avoid /dist/ imports from sdk-ui-ext.

---

Supported PR commands:

| Command                  | Description            |
| ------------------------ | ---------------------- |
| `ok to test`             | Re-run standard checks |
| `extended test`          | BackstopJS tests       |
| `extended check sonar`   | SonarQube tests        |
| `extended check cypress` | Cypress E2E tests      |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [x] `check` passes
-   [x] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by no23reason** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/5497/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/7215/ : SUCCESS in 17m 33s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/5516/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/7234/ : SUCCESS in 15m 24s

**Comment by no23reason** [comment]
extended check cypress

File: common/config/rush/experiments.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
    * to temporarily restore the old behavior where everything must be rebuilt whenever pnpm-lock.json
    * is modified.
    */
-  "legacyIncrementalBuildDependencyDetection": true,
+  // "legacyIncrementalBuildDependencyDetection": true,
```
Comment (plavemar) [inline common/config/rush/experiments.json:15]: Is there any reason why this is commented out instead of set up to false?

**Pull request merged**